api/src/main/java/com/velocitypowered/api/command/BrigadierCommand.java

```diff
@@ -0,0 +1,46 @@
+package com.velocitypowered.api.command;
+
+import com.mojang.brigadier.builder.ArgumentBuilder;
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.mojang.brigadier.tree.CommandNode;
+
+/**
+ * A command that uses Brigadier for parsing the command and providing
+ * suggestions to the client.
+ */
+public interface BrigadierCommand extends Command<BrigadierCommandInvocation> {
+
+  /**
+   * Returns an {@link ArgumentBuilder} used to specify the structure of
```

Comment:
I would like to see some better description on why this is the case. This isn't obvious on first glance.

Reply:
This is a helper method, callers can still use the longer `LiteralArgumentBuilder.<CommandSource>.literal(alias)`. Most examples I could find of Brigadier commands prefer to use the statically imported `literal()`, but it's not possible in our case since it requires to specify the `<CommandSource>` type.